brainframe/cli/translations/general.en.yml

```diff
@@ -16,3 +16,15 @@ en:
   staging-missing-credentials: "The %{username_env_var} and %{password_env_var}
   variables must be set to access staging."
   user-not-root: "This command must be run as root!"
+  docker-bad-permissions: "Your user does not have sufficient privileges to
+  use Docker."
+  restart-for-group-access: "Your user has been added to the \"%{group}\"
+  group but you must restart to apply this change. Alternatively, you can try
+  again with sudo."
```

Comment:
Other messages use the phrase "as root" (which is technically more accurate). Do we want to stay consistent with these messages as well?